request_profiler/models.py

```diff
@@ -153,6 +153,10 @@ class ProfilingRecord(models.Model):
     request_uri = models.URLField(
         verbose_name="Request path"
     )
+    query_string = models.TextField(
+        null=True,
```

Comment:
If we allow null to be True, we will have cases in which `query_string ` will be null and cases in which `query_string` will be an empty string, and since django treats these both the same, it would be much cleaner if we used only one of them, like
```suggestion
        null=False,
        blank=True
```